docs/modules/ROOT/pages/tutorial-onpremise.adoc

```diff
@@ -101,37 +111,111 @@ To add Couchbase Server end points, create a similar JSON format file to the lin
 
-==== Alerting rules
+==== Loki
+Configuration file for log aggregation system Loki is present on `/etc/loki/local-config.yaml` inside the container. Update the environment variable to change the config file location or update the config file to change property of Loki accordingly.
+
+[source, console]
+----
+# Loki configuration
+LOKI_CONFIG_FILE=${LOKI_CONFIG_FILE:-/etc/loki/local-config.yaml}
+----
+
+==== Grafana
+To configure Grafana refer to <<configure_grafana>> section.
+
+==== Alert
+To configure Alerting refer to <<configure_alert>> section.
+
+=== Accessing The Deployed Services
+There are 2 ways to access different services available in the deployed CMOS stack.
+
+==== Using Nginx Service
+A landing page provided by the CMOS stack through Nginx. It has links to all the deployed services. You can navigate to any service from here. To get the port of your host machine use this:
+[source, console]
+----
+$ docker container port CMOS 8080
+----
+----
+0.0.0.0:49278
+:::49278
+----
+From host machine open http://0.0.0.0:49278 to access it.
+
+==== Direct Use Of Port Of Virtual Machine
+Use `docker ps` or `docker inspect X` to see the local ports exposed, there are multiple services running inside the container and each service has it's own port. For example, the mapping to `3000` is the Grafana, to get the port of the host machine use this:
+
+[source, console]
+----
+$ docker container port CMOS 3000
+----
+----
+0.0.0.0:49283
+:::49283
+----
+Then from host machine open http://0.0.0.0:49283 to access Grafana. Same applies for other services.
+
+=== Connecting Couchbase Cluster
+
+You can connect a couchbase cluster using API call as well as add cluster option given on the UI. Use this API call to connect a couchbase cluster:
+
+----
+CLUSTER_MONITOR_USER=admin
+CLUSTER_MONITOR_PWD=password
+CLUSTER_MONITOR_ENDPOINT=http://localhost:$(docker container port CMOS 7196)
+COUCHBASE_USER=Administrator
+COUCHBASE_PWD=password
+COUCHBASE_ENDPOINT=http://<hostname/IP>:8091
+
+# Make the api call via curl. You can use any other tool to make this API call
+curl -u "${CLUSTER_MONITOR_USER}:${CLUSTER_MONITOR_PWD}" -X POST -d '{ "user": "'"${COUCHBASE_USER}"'", "password": "'"${COUCHBASE_PWD}"'", "host": "'"${COUCHBASE_ENDPOINT}"'" }' "${CLUSTER_MONITOR_ENDPOINT}/api/v1/clusters"
+----
+
+== Configure Grafana [[configure_grafana]]
+Various dashboard to monitor couchbase cluster is shipped out of the box in Grafana. Connect a couchbase cluster and navigate to Grafana to see the graphs. You can list all the dashboards using the https://grafana.com/docs/grafana/latest/dashboards/search/#dashboard-search[search dashboard option^]. You may create https://grafana.com/docs/grafana/latest/getting-started/getting-started/#step-3-create-a-dashboard[additional dashboards^] as per your needs.
+
```

Comment:
@shovanmaity can you add a screen shot here. Saying something like:
```
Post complete setup you should be able to see metrics in your grafana dashboard:
<Attach a screenshot here>
```

And the same for Alerting rules, we need a screenshot of alert manager page.

Reply:
done